src/main/java/com/google/api/codegen/php/PhpGapicContext.java

```diff
@@ -132,8 +132,10 @@ public String addImport(String typeName) {
    * Returns the PHP representation of a reference to a type.
    */
   public String typeName(TypeRef type) {
-    if (type.isMap() || type.isRepeated()) {
+    if (type.isMap()) {
       return "array";
+    } else if (type.isRepeated()) {
+      return basicTypeName(type) + "[]";
```

Comment:
I was referring to: https://github.com/google/google-api-php-client-services/blob/master/Translate/Resource/Detections.php

But as long as this is recommended by the gcloud-php, I am okay.